katello-pull-transport-migrate

```diff
@@ -1,19 +1,17 @@
-#!/bin/sh
-
-# set SBINDIR to /sbin if it is not set
-if [ -z "$SBINDIR" ]; then
-    SBINDIR=/usr/sbin
-fi
+#!/bin/bash
```

Comment:
Do we rely on any bash specific behavior?

Reply:
AFAIK `set -e` is a bashism.

Reply:
You may be right, at least checkbashisms on Debian 10 doesn't find anything. Perhaps @evgeni can tell me I'm wrong.